### ISSUE TITLE
Refactor RESET MODIFICATIONS and understand  special selectors

### ISSUE THREAD

**Issue by lifecrisis** [lifecrisis/homepage#14]
I ended up removing the dotted outline in issue #6. This lead to a lack of visual indicators of the active/focused/hovered link for people with accessibility issues. So, see #6.

Do research to understand:
* ":hover"
* ":active"
* ":focus"

These selectors need to be used properly, even when the dotted outline is removed. They serve a purpose!
